traits-stubs/traits-stubs/trait_type.pyi

```diff
@@ -14,6 +14,11 @@ from .base_trait_handler import BaseTraitHandler as BaseTraitHandler
 
 trait_types: Dict[str, int]
 
+
+class NoDefaultSpecified:
```

Comment:
`NoDefaultSpecified` is not a class. Should this be renamed too?

Reply:
Ah yes; that should be fixed. Not by renaming, though - the stub should be for the `NoDefaultSpecified` value, not for the class.

Reply:
I confess that I'm not actually sure what the right fix is, though.

Reply:
Given that `NoDefaultSpecified` is just a constant value, so that there are no methods, return types, etc. to record, the right fix may be simply to remove this entry from the stub. @kitchoi Opinions?

Reply:
Removing sounds good to me.

Reply:
Done.